packages/hint/src/lib/utils/resource-loader.ts

```diff
@@ -293,21 +315,27 @@ const generateConfigPathsToResources = (configurations: string[], name: string,
  *
  * 1. core resource
  * 2. `@hint/` scoped package
- * 3. `hint-` prefixed package
+ * 3. `webhint-` prefixed package
  * 4. external hints
  *
  */
 export const loadResource = (name: string, type: ResourceType, configurations: string[] = [], verifyVersion = false) => {
     debug(`Searching ${name}…`);
     const isSource = fs.existsSync(name); // eslint-disable-line no-sync
+    const isPackage = isFullPackageName(name, type);
     const nameSplitted = name.split('/');
 
-    const packageName = nameSplitted[0];
+    let scope = '';
+
+    if (isPackage && nameSplitted[0].startsWith('@')) {
+        scope = `${nameSplitted.shift()!}/`;
```

Comment:
Add comment explaining that this removes the scope so `nameSplitted[0]` will point to the non-scoped package name afterward (or refactor to avoid mutating the array).